wrappers/systemd.template.service.in

```diff
@@ -4,7 +4,7 @@
 [Unit]
 Description=@capbrand@ Directory Server %i.
 PartOf=@systemdgroupname@
-After=chronyd.service ntpd.service network-online.target
+After=chronyd.service ntpd.service network-online.target systemd-tmpfiles-setup.service
```

Comment:
I think it should be wants as well sinc after is just ordering but wants will force it to be started

Reply:
Right, systemd man page says it's a common pattern to add both: https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Before=

Fixed.